### ISSUE TITLE
Query for attributes containing '-' fails

### ISSUE THREAD

**Issue by martin-macak** [AntonShuvaev/intellij-dynamodb#3]
## Description

When query contains attribute with `-` in name then it fails with following error:

```
ValidationException
Invalid KeyConditionExpression: Syntax error; token: "-", near: "GI-1"
```

This is caused by not masking ExpressionAttributeNames with valid values. This is btw. the same error that competing product Dynobase has. This is killer, because it makes plugin unusable for most of tables that follow this naming convention. It's extremely inconvenient as AWS encourages developers to name their entities and attributes in DNS compliant way, which contains `-` character.

## Steps to reproduce

- Create table with GI named `GI-1-PK`
- Scan for table in DynamoDB plugin
- Fill in query `GI-1-PK = "A"`

## Suggestion for fix

Use `ExpressionAttributeNames` to rename all attributes to safe names

**Comment by martin-macak** [comment]
Seems like query is constructed like

```
Query
{
  "Limit": 50,
  "KeyConditionExpression": "GI-1-PK = :partitionKeyVal",
  "ExpressionAttributeValues": {
    ":partitionKeyVal": {
      "S": "A"
    }
  },
  "IndexName": "GI-1",
  "TableName": "VerticeUserManagement-dev-v1"
}
```

So it should be super easy to add `ExpressionAttributeNames`

**Comment by AntonShuvaev** [comment]
Thanks for reporting the issue. I will fix it as soon as possible.

**Comment by AntonShuvaev** [comment]
Fixed in 2022.1.6
